### ISSUE TITLE
Tasks are always visible (2)

### ISSUE THREAD

**Issue by geoffrey-vl** [actboy168/vscode-tasks#7]
With vscode  (1.28) and version 0.2.4 of this extension installed, tasks are always visible on status bar regardless of the "hide" setting en tasks.json. I'm on Windows, the tasks.json file is subfolder .vscode and we're opening the project as a folder, not as a workspace.

Related to https://github.com/actboy168/vscode-tasks/issues/5.

**Comment by actboy168** [comment]
Please provide your tasks.json.

**Comment by geoffrey-vl** [comment]
Sorry, off course I've should have added that in the first place. Here we go:

```{
    // See https://go.microsoft.com/fwlink/?LinkId=733558
    // for the documentation about the tasks.json format
    "version": "2.0.0",
    "tasks": [
        {
            "label": "mbed build",
            "type": "shell",
            "command": "mbed compile",
            "args": [
                "--profile=debug",
                "-t",
                "GCC_ARM",
                "-m",
                "NUCLEO_F429ZI"
            ],
            "group": {
                "kind": "build",
                "isDefault": true
            },
            "problemMatcher": [
                "$gcc"
            ]
        },
        {
            "label": "mbed clean",
            "type": "shell",
            "command": "echo cleaning BUILD... & RMDIR BUILD /S /Q & echo Done! BUILD directory empty",
            "options": {
                "statusbar": "hide"
            }
        },
        {
            "label": "startOpenOcdServer",
            "type": "shell",
            "command": "openocd.exe -f board\\st_nucleo_f4.cfg",
            "isBackground": true,
            "problemMatcher": {
                "owner": "custom",
                "pattern": {
                    "regexp": "_____"
                },
                "background": {
                    "activeOnStart": true,
                    "beginsPattern": "^.*GNU MCU Eclipse 64-bits Open On-Chip Debugger*$",
                    "endsPattern": "^.*Info : Listening on port 3333 for gdb connections*$"
                }
            },
            "options": {
                "statusbar": "hide"
            }
        },
        {
            "label": "stopOpenOcdServer",
            "type": "shell",
            "command": "Taskkill /IM openocd.exe /F",
            "options": {
                "statusbar": "hide"
            }
        }
    ]
}```

**Comment by actboy168** [comment]
I tested it in VSCode 1.35.1 and it works fine. I noticed that you are using VSCode1.28, maybe you should upgrade your VSCode.

**Comment by geoffrey-vl** [comment]
Well, I'm sorry but I'm kind of stuck on that version of vscode, due to how Tasks behave in newer versions. I understand that you don't have any plans to support earlier versions.

**Comment by actboy168** [comment]
You can roll back the extension to a usable version, if the rollback is already available in VSCode 1.28.
